npm/ng-packs/apps/dev-app/src/app/home/home.module.ts

```diff
@@ -2,9 +2,10 @@ import { NgModule } from '@angular/core';
 import { SharedModule } from '../shared/shared.module';
 import { HomeRoutingModule } from './home-routing.module';
 import { HomeComponent } from './home.component';
+import { LoadingDirective, ThemeSharedModule } from '@abp/ng.theme.shared';
 
 @NgModule({
   declarations: [HomeComponent],
-  imports: [SharedModule, HomeRoutingModule],
+  imports: [SharedModule, HomeRoutingModule, ThemeSharedModule],
```

Comment:
`ThemeSharedModule` has been import in `SharedModule`
```suggestion
  imports: [SharedModule, HomeRoutingModule],
```

Reply:
Thanks you for your feedbacks, i will fix it.